src/java/fr/paris/lutece/portal/service/dashboard/admin/AdminDashboardService.java

```diff
@@ -189,7 +189,7 @@ public void doMoveDashboard( IAdminDashboardComponent dashboard, int nOldColumn,
         {
             if ( AppLogService.isDebugEnabled( ) )
             {
```

Comment:
It is unnecessary to check if logging is enabled because it is already managed with the debug level.